### PR TITLE
add storageversionmigrations to related objects

### PR DESCRIPTION
**Pull request by sanchezl**
Add storageversionmigrations to related objects

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -84,6 +84,7 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		"kube-storage-version-migrator",
 		[]configv1.ObjectReference{
 			{Group: "operator.openshift.io", Resource: "kubestorageversionmigrators", Name: "cluster"},
```
Comment (deads2k) [inline pkg/operator/starter.go:86]: these should also be in the clusteroperator yaml file for the CVO.  That way we get the early reporting if we crash.

Comment (sanchezl) [reply pkg/operator/starter.go:86]: @deads2k Done.

**Pull request merged**